Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs

```diff
@@ -25,7 +25,13 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 		void SetupLayer()
 		{
-			Layer.CornerRadius = 5;
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
+				cornerRadius = 5f; // default corner radius
```

Comment:
Equality comparison between floating point numbers is not a good idea anyway. It should be changed to **if (cornerRadius < 0f)** IMHO.